man/runc-list.8.md

```diff
@@ -4,12 +4,9 @@
 # SYNOPSIS
    runc list [command options] [arguments...]
 
-# DESCRIPTION
-   The default format is table.  The following will output the list of containers
-in json format:
-
-    # runc list -f json
-
 # OPTIONS
    --format, -f         select one of: table or json.
+                        The default format is table. The following will output
+                        the list of containers in json format:
+                        # runc list -f json
```

Comment:
And maybe we also need to remove the example in code's --help option, as in corrent code:

# ./runc list --help
...
OPTIONS:
   --format, -f         select one of: table or json.

The default format is table.  The following will output the list of containers
in json format:

    # runc list -f json
   --quiet, -q  display only container IDs
#  

What is your opinion?